swarm/tools/runs_gc.py

```
@@ -58,32 +58,43 @@ class RunInfo:
+        if self._size_bytes is None:
+            import os
 
-def get_dir_size(path: Path) -> int:
-    """Get total size of a directory in bytes."""
-    total = 0
-    try:
-        for entry in path.rglob("*"):
-            if entry.is_file():
+            total = 0
+
+            def _scan(p):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `_scan`

Add return type annotation: `None`

(ANN202)